packages/custom-node-generator/package.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "rn-audioapi-custom-node-generator",
+    "version": "0.0.2",
+    "description": "A cli generator for custom node used by react-native-audio-api",
+    "main": "lib/generator.js",
+    "bin": {
+      "my-generator": "./bin/cli.js"
+    },
+    "author": "michal.dydek@swmansion.com",
```

Comment:
Can we get access to the npm package (unless you want to pulish it under different name)